problemsets-mat223/modules/module8-exercises.tex

```diff
@@ -46,6 +46,51 @@
 						F}$.
 				\end{enumerate}
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item 
+					\begin{enumerate}
+						\item $[\vec{u}]_{\mathcal{E}} = \mat{1\\8}$ \hspace{4mm} $[\vec{v}]_{\mathcal{E}} = \mat{-1\\3}$ \hspace{4mm} $
```

Comment:
`$` shouldn't end a line. Also, no manual spacing with `\hspace`. Use a comma or a word to separate a list of answers.